install-tests/react-native/listen.js

```diff
@@ -0,0 +1,43 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2021 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+const { createServer } = require("http");
+const PORT = 3000;
+const EXPECTED_MESSAGE = "Persons are Alice, Bob, Charlie";
```

Comment:
I think `EXPECTED_MESSAGE` should be an argument to `listen.js`.

Reply:
How come? I wouldn't expect any other app to use this script. If another app comes along, that would send a different message I would be okay about taking this as an argument. In its current form, its easier to just `node listen.js` when debugging this - as you don't have to remember what message to provide as an argument.

Reply:
Yeah that's a good point.   If we listen for multiple things later on, then maybe it would make sense to break it into arguments, but might as well just leave it explicit for now.